proposals/0001-simd-process.md

```diff
@@ -83,6 +84,7 @@ The stages in a lifecycle of a proposal are as follows:
 - Stagnant
 - Withdrawn
 - Implemented
+- Activated
 
 ```mermaid
 flowchart LR
```

Comment:
seems like the `Idea` state below dies with the gh discussions change?

Reply:
Yeah, I think that's a reasonable point. Considering making a change to the flow further down the line.